demos/service-workflow/src/worker.ts

```diff
@@ -5,7 +5,7 @@ async function run() {
   const worker = await Worker.create({
     workflowsPath: require.resolve('./workflows'),
     activities,
-    taskQueue: 'translations-tasks',
+    taskQueue: 'translation-tasks',
```

Comment:
this resulted in no workers polling the queue that the workflows were started on, which meant the workflows never finished. we could help users avoid this mistake by defining it in one place and importing